File: Sasta Shark Tank.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b;
        scanf("%d%d",&a,&b);
        if(a*10>b*5)
        printf("FIRST\n");
        else if(a*10<b*5)
        printf("SECOND\n");
        else
        printf("ANY\n");
    }
}

File: Sasta Shark Tank.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            if(a*10>b*5)
            System.out.println("FIRST");
            else if(a*10<b*5)
            System.out.println("SECOND");
            else
            System.out.println("ANY");
        }
    }
}